Process.c

```diff
@@ -150,11 +150,15 @@ void Process_printTime(RichString* str, unsigned long long totalHundredths) {
    unsigned long long totalSeconds = totalHundredths / 100;
 
    unsigned long long hours = totalSeconds / 3600;
+   unsigned long long days = totalSeconds / (24 * 3600);
```

Comment:
Prefer explicit `86400` and avoid parens.

Reply:
Okay. I'll fix that but may I ask why? Constant folding is something compilers are really good at.

Reply:
Two reasons: First is, that for consistency you'd otherwise also have to have the `3600` be written as `60 * 60`. Second is, that by using the `86400` as a single constant you avoid the additional parens that make the code at that place look unnecessarily complex when that value is a quite commonly used value.

Thus even though constant folding in compilers has become really good in recent years™, there's no additional value gained from it in this case. You could BTW also have used the value from `HOURS` in your calculation instead, if the emphasis really would have needed to be on the day having 24 hours.